src/Components/Server/src/Circuits/DefaultPersistedCircuitCache.cs

```diff
@@ -0,0 +1,169 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Components.Server.Circuits;
+
+// Default implmentation of ICircuitPersistenceProvider that uses an in-memory cache
+internal sealed partial class DefaultInMemoryCircuitPersistenceProvider : ICircuitPersistenceProvider
```

Comment:
Should this file name be changed to match the name of this class?

Reply:
Yeah, this was left over from the prototype